src/views/virtualmachines/list/VirtualMachinesList.tsx

```diff
@@ -130,7 +129,7 @@ const VirtualMachinesList: FC<VirtualMachinesListProps> = ({ kind, namespace })
     }));
   };
 
-  const [columns, activeColumns] = useVirtualMachineColumns(namespace, pagination, data);
+  const [_, activeColumns] = useVirtualMachineColumns(namespace, pagination, data);
```

Comment:
why nit needed anymore and only active?

Reply:
unused variable. Eslint has a problem with that. 
When we'll add the columnManagementModal in the ListPageFilter, we'll use it back again.

Reply:
yes i know ive added the linter :) 
I wanted to know why it isn't in use, u answered :)